src/styles/theme.ts

```diff
@@ -1,4 +1,6 @@
-export const theme = {
+import { TColor } from '@/types';
+
+export const theme: { [key: string]: TColor } = {
```

Comment:
여기서 타입을 { [key: string]: TColor }로 선언해버리면 `theme`객체에 대한 속성을 지정할 수 없게 됩니다.
그렇게 되면 저희가 사용할 때 `theme`객체에서 속성에 대한 자동완성을 사용할 수 없게 됩니다!

```suggestion
export const theme = {
// ...
} satisfies { [key: string]: TColor };
```

로 변경하시면 원하는 타입 작업과 속성 지정이 가능할 것 같습니다!

Reply:
타입을 저렇게 선언하면 자동완성이 안된다는 사실을 생각 못했네요... `satisfies` 로 수정하겠습니다!

Reply:
`satisfies`에 대해서 배우고 갑니다..